canvas_modules/common-canvas/src/common-canvas/svg-canvas-renderer.js

```diff
@@ -2985,6 +2993,57 @@ export default class SVGCanvasRenderer {
 		this.logger.logEndTimer("displayNodes " + this.getFlags());
 	}
 
+	displayEditIcon(spanObj, node) {
```

Comment:
Is there a better way to handle this situation?
<img width="230" alt="image" src="https://user-images.githubusercontent.com/1628100/108123242-66438280-705a-11eb-8b8d-2e23c1ddc12f.png">

Maybe some auto wrapping of the text to the next line for preview?

Reply:
This problem only occurs when the 'show full label on hover' option is switched on .  I fixed the code so the edit icon appears at the end of the hover over text.